ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/RdfFormatter.java

```diff
@@ -0,0 +1,53 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter;
+
+import com.github.jsonldjava.core.JsonLdOptions;
+import org.apache.jena.atlas.json.JsonObject;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.JsonLDWriteContext;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFFormat;
+import org.apache.jena.riot.RDFWriter;
+import org.apache.jena.riot.writer.JsonLD10Writer;
+import org.apache.jena.sparql.util.Context;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter.PrefixAdder.*;
+
+public class RdfFormatter {
```

Comment:
My personal preference would have been a strategy pattern instead of a public static method. This kind of class has a tendency to grow with more if statements for different Langs.

Now you are injecting a "lang" and a "type" in most outputters. It could be cleaner for the autoConfigs to create the correct writer class and inject that in the output components. Then every output component expects an 'RdfWriter' interface without caring about langs or frametypes.